### PR TITLE
Spell check and grammar changes to docs/config/*.md

### PR DESCRIPTION
**Pull request by uberhacker**
- [x] My code includes the latest code from the `master` branch
- [x] My code includes an update to the [CHANGELOG](https://github.com/lando/lando/tree/master/docs/help)
- [ ] My code includes a [functional test](https://docs.lando.dev/contrib/contrib-testing.html) if applicable
- [ ] My code includes a [unit test](https://docs.lando.dev/contrib/contrib-testing.html) if applicable
- [x] My code includes documentation updates if applicable.
- [ ] My code passes relevant CI status checks
- [ ] I have pinged [a project committer](https://docs.devwithlando.io/contrib/contributing.html#committers) when I think my code is ready for prime time.

### CODE REVIEW

File: docs/help/2020-changelog.md [hunk 1]
```diff
@@ -19,7 +19,6 @@ Otherwise, scroll down and use the first release that has the green `Latest Rele
 * Fixed bug causing app start to fail if proxy was unable to locate suitable ports [#1857](https://github.com/lando/lando/pull/1857)
 * Fixed bug causing `ssh` to fail when username or keypaths had spaces in them [#2031](https://github.com/lando/lando/pull/2031)
 * Improved spelling and grammar in docs, code and comments, thanks @uberhacker!
-* Spell check docs/basics/*.md
```
Comment (ccjjmartin) [inline docs/help/2020-changelog.md:22]: Should this be removed?

File: docs/config/apache.md [hunk 2]
```diff
@@ -16,24 +16,24 @@ description: Add an Apache webserver service to your Lando config for local deve
...
+To use a patch version, you can do something as shown below:
 
 ```yaml
 services:
   my-service:
     type: apache:2.4.33
 ```
 
-But make sure you use one of the available [patch tags](https://hub.docker.com/r/bitnami/apache/tags) for the underlying image we are using.
+But make sure you use one of the available [patch tags](https://hub.docker.com/r/bitnami/apache/tags) for our underlying image.
```
Comment (pirog) [inline docs/config/apache.md:30]: Our is not accurate. Most the images are not "our" images and its important we do not communicate that they are.

**Comment by pirog** [comment]
@uberhacker i left a few comments as well but overall lots of improvements!

@serundeputy might have additional thoughts so id be interested in his review

File: docs/config/apache.md (hunk 2 above)
Comment (uberhacker) [reply docs/config/apache.md]: Thanks for the feedback.  Fixed.

**Pull request closed without merging**